test/queryClient.test.ts

```diff
@@ -2,9 +2,35 @@
 // SPDX-License-Identifier: Apache-2.0
 
 import { GraphqlQueryClient } from '../packages/network-clients';
-import { NETWORK_CONFIGS } from '../packages/network-clients';
+import { NETWORK_CONFIGS } from '../packages/network-clients/src';
 import assert from 'assert';
-import { GET_INDEXER } from '../packages/network-clients';
+import { GET_INDEXER } from "../packages/network-clients/src/graphql/indexers";
+import { GET_INDEXERS } from "../packages/network-clients/src/graphql/indexers";
+import {
+  GET_ALL_DELEGATIONS,
+  GET_DELEGATION,
+  GET_DELEGATOR,
+  GET_INDEXER_DELEGATORS
+} from "@subql/network-clients/dist/graphql/delegations";
```

Comment:
Can import all these queries from `@subql/network-clients` directly now

Reply:
when i change to  `from @subql/network-clients`, all queries cannot be founded